src/utils/wallet.js

```diff
@@ -637,7 +644,7 @@ class Wallet {
         return availableKeys;
     }
 
-    async getAccountBasic(accountId) {
+    getAccountBasic(accountId) {
```

Comment:
Please find all usages of this method and remove the `await` keyword for clarity and consistency, and also make sure that there isn't anywhere that is using `getAccountBasic(xxxx).then(...)`, since that will be a breaking usage of the method.  It might seem safe to leave `await` keywords on usages of the function, but if we target any environment without native `async/await` support, these will be transpiled down to Promise usage...and when that happens, the transpiled code may break because it will be converted to `getAccountBasic(xxxx).then(...)` but `getAccountBasic()` won't return a Promise because it's no longer marked `async`.

Also just a note, this type of change will need to be treated as a breaking change in the future when we are exporting `Wallet` for usage by other apps/consumers, because  we have changed the function return type from `a Promise that resolves to a near-api-js Account object` to `a near-api-js Account object`

Reply:
Thanks, I removed it at some point and totally forgot about adjusting calls.

> Also just a note, this type of change will need to be treated as a breaking change in the future

Right, roger that.